src/asn1.ml

```diff
@@ -615,11 +667,263 @@ struct
     map f g @@ sequence2
       (required ~label:"algorithm" Algo.grammar)
       (required ~label:"parameters" DH.Params.grammar)
+
 end
 
 let map_result f = function Result.Ok x -> Result.Ok (f x) | Result.Error _ as r -> r
 let default_result default = function Result.Error _ -> default () | Result.Ok _ as r -> r
 
+module CVC = struct
+  let rsa_oid = Asn.OID.of_string "0.4.0.127.0.7.2.2.2.1.4"
```

Comment:
I'm fine with the `of_string` solution, but here are a couple benefits in doing otherwise:

- that documents the name of the root OID
- that removes a bit of duplication (not super important but typos happen)
- this changes from running a parser at runtime to using a constructor. That is the same difference as writing `Int64.of_string "12345"` vs writing `12345L`. With a local open, there's little danger in using custom operator (`Asn_oid.(id_ta_terminal_authentication <|| [1;4])` is almost an OID litteral).